scrapylib/hcf.py

```diff
@@ -211,12 +223,6 @@ def _get_new_requests(self):
         self._msg('Read %d new batches from slot(%s)' % (num_batches, self.hs_consume_from_slot))
         self._msg('Read %d new links from slot(%s)' % (num_links, self.hs_consume_from_slot))
 
-    def _save_new_links_count(self):
-        """ Save the new extracted links into the HCF."""
-        for slot, new_links in self.new_links.items():
-            self._msg('Stored %d new links in slot(%s)' % (len(new_links), slot))
```

Comment:
Previously `new_links` is cleaned in `close_spider`. However I do not think that is necessary since the spider is closing, everything here would be cleaned once the instance of.`HcfMiddleware` is recycled.